documentation/modules/security/proc-accessing-kafka-using-routes.adoc

```diff
@@ -5,18 +5,39 @@
 [id='proc-accessing-kafka-using-routes-{context}']
 = Accessing Kafka using OpenShift routes
 
-This procedure describes how to access a Strimzi Kafka cluster from an external client outside of OpenShift using routes.
+[role="_abstract"]
+Use OpenShift routes to access a Strimzi Kafka cluster from clients outside the OpenShift cluster.
+Routes support connections from from HTTP and HTTPS-based applications.
```

Comment:
Again, this is confusing if not directly followed by the TLS passthrough explanation.

Reply:
Removed